packages/drivers/puppeteer/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "@pilot/puppeterr-driver",
+    "version": "0.0.1",
+    "description": "Puppeterr driver for pilot and copilot usage",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+      "dist"
+    ],
+    "publishConfig": {
+      "registry": "https://registry.npmjs.org"
+    },
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/wix-incubator/detox-copilot.git"
+    },
+    "scripts": {
+      "build": "tsc && tsc-alias",
+      "test": "echo No tests avialable for this package",
+      "bump-version": "npm version patch && git push && git push --tags",
+      "release": "npm run test && npm run bump-version && npm run build && npm publish",
+      "lint": "eslint . --ext .ts",
+      "lint:fix": "eslint . --ext .ts --fix",
+      "type-check": "tsc --noEmit"
+    },
+    "bugs": {
+      "url": "https://github.com/wix-incubator/detox-copilot/issues"
+    },
+
+    "dependencies": {
+     "@pilot/core": "*"
```

Comment:
seems like this will keep the `"*"` on the generated package, so whoever installs the package later could end up with the newest available version of that dependency at their install time.. even if it's broken.

Reply:
seems like we'll need a semver here